tests/test_rank.py

```diff
@@ -378,3 +380,47 @@ def test_unsupported_method_for_adjust_pred_probs():
         method = "confidence_weighted_entropy"
 
         _ = rank.get_label_quality_scores(labels, pred_probs, adjust_pred_probs=True, method=method)
+
+
+def test_find_top_issues():
+    X_train = data["X_train"]
+    X_test = data["X_test"]
+    X_ood = np.array([[999999999.0, 999999999.0]])  # Create OOD datapoint
+    X_test_with_ood = np.vstack([X_test, X_ood])  # Add OOD datapoint to X_test
+
+    # Create OOD object (use knn without cosine metric to identify X_ood correctly)
+    knn = NearestNeighbors(n_neighbors=5).fit(X_train)
+    ood_outlier = outlier.OutOfDistribution(params={"knn": knn})
+    ood_scores = ood_outlier.score(features=X_test_with_ood)
+
+    # Get top ood score for outlier example
+    top_outlier_indices = rank.find_top_issues(quality_scores=ood_scores, top=len(ood_scores))
+    top_outlier_indices_more_k = rank.find_top_issues(quality_scores=ood_scores, top=100000)
```

Comment:
```suggestion
    top_outlier_indices_more_k = rank.find_top_issues(ood_scores, top=100000)
```

Reply:
maybe also good to sometimes test positional args are truly positional as I suggested above, now that we commonly use `,*,` in the function signatures